docs/EspressifIDE.md

```diff
@@ -48,3 +48,17 @@ After un-archiving, if the application still complains, check/remove the attribu
 ```
 $ xattr -dr com.apple.quarantine ~/Downloads/Espressif-IDE.app
 ```
+### Espressif-IDE v2.4.0 Installer for Windows
+Espressif-IDE Installer (All-in-one) is an offline installer and it comes with all the required dependencies to work with the ESP-IDF Application development.
+
+Installer includes:
+- Embedded Python
+- Cross-compilers
+- OpenOCD
+- CMake and Ninja build tools
+- ESP-IDF
+- Espressif-IDE (includes essential Eclipse CDT Packages, IDF Eclipse Plugins and other third-party plugins required)
+- Amazon Corretto OpenJDK
```

Comment:
The installer also deploys latest stable ESP-IDF, it's part of deployment process so that people behind corporate firewalls can use whole solution out-of-box. It might be worthy to mention it in the list.

Reply:
Ok @georgik Let me add that